app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -110,6 +116,7 @@
 }
 
 .mortgagecalc__spread {
+  margin-bottom: 0.5rem;
```

Comment:
The rest of the Mortgage Calculator doesn't use this yet but I think it's a good move, so I've changed it to use this variable now.